docker/Dockerfile.mediamtx

```diff
@@ -1,12 +1,19 @@
-FROM ubuntu:20.04
+FROM ubuntu:24.04
 
-RUN apt update
-RUN apt install -y curl
+ARG TARGETARCH
+ENV MEDIAMTX_VERSION="v1.9.3"
 
-RUN curl -L https://github.com/bluenviron/mediamtx/releases/download/v1.9.3/mediamtx_v1.9.3_linux_amd64.tar.gz -o /mediamtx.tar.gz
-RUN tar xzvf /mediamtx.tar.gz
-RUN rm /mediamtx.tar.gz
+RUN apt-get update \
+    && apt-get install -y \
+        ca-certificates \
+        curl \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
-COPY docker/mediamtx.yml /mediamtx.yml
+RUN curl -L https://github.com/bluenviron/mediamtx/releases/download/${MEDIAMTX_VERSION}/mediamtx_${MEDIAMTX_VERSION}_linux_amd64.tar.gz -o /mediamtx.tar.gz \
+    && tar xzvf /mediamtx.tar.gz \
+    && rm /mediamtx.tar.gz
```

Comment:
@mjh1  `mediamtx.tar.gz` contains `mediamtx.yml` file and it looks it differs from the one in [our repo](https://github.com/livepeer/go-livepeer/blob/master/docker/mediamtx.yml) with these lines:
```diff
< logLevel: info
---
> logLevel: debug

<   runOnReady:
---
>   runOnReady: curl http://localhost:5936/live/video-to-video/$MTX_PATH/start -F callback=$HOSTNAME -F source_id=$MTX_SOURCE_ID -F source_type=$MTX_SOURCE_TYPE -F query=$MTX_QUERY
```

Could we use [environment variables](https://github.com/bluenviron/mediamtx?tab=readme-ov-file#configuration) for overriding the defaults instead? Like:
```
ENV MTX_LOGLEVEL=debug
ENV MTX_PATHDEFAULTS='curl http://localhost:5936/live/video-to-video/$MTX_PATH/start -F callback=$HOSTNAME -F source_id=$MTX_SOURCE_ID -F source_type=$MTX_SOURCE_TYPE -F query=$MTX_QUERY'
```

Reply:
https://github.com/livepeer/livepeer-infra/pull/2478/files#r1850383090

Reply:
ohh nice spot 👍